babel/src/main/java/org/apache/calcite/sql/babel/SqlHostVariable.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+/**
+ * A {@code SqlHostVariable} is a host variable of the form ":name".
+ */
+public class SqlHostVariable extends SqlIdentifier {
+
+  /**
+   * Creates a {@code SqlHostVariable}.
+   *
+   * @param name  Name of the host variable
+   * @param pos  Parser position, must not be null
+   */
+  public SqlHostVariable(String name, SqlParserPos pos) {
+    super(name, pos);
+  }
+
+  @Override public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
+    writer.print(":");
+    writer.print(names.get(0));
```

Comment:
nit: Can these two lines be combined into one?

Reply:
Yeah good point, I updated it